### PR TITLE
customize submariner subscription

### PR DESCRIPTION
**Pull request by skeeey**
Signed-off-by: liuwei <liuweixa@redhat.com>

### CODE REVIEW

**Comment by skeeey** [comment]
see https://github.com/open-cluster-management/backlog/issues/8683

File: pkg/apis/submarinerconfig/v1alpha1/types.go [hunk 1]
```diff
@@ -46,12 +46,46 @@ type SubmarinerConfigSpec struct {
...
+
+	// CatalogSourceNamespace represents the sourceNamespace of a submariner subscription.
+	// The default value is openshift-marketplace
+	// +optional
+	CatalogSourceNamespace string `json:"sourceNamespace,omitempty"`
+
+	// Name represents the name of a submariner subscription.
+	// The default value is submariner
+	// +optional
+	Name string `json:"name,omitempty"`
```
Comment (qiujian16) [inline pkg/apis/submarinerconfig/v1alpha1/types.go:71]: will we have a different  name?

Comment (skeeey) [reply pkg/apis/submarinerconfig/v1alpha1/types.go:71]: only one case, if I create a CatalogSource by myself, I can change the name...

**Pull request merged**